golden/runner.py

```
@@ -415,18 +400,29 @@ def _effective_us(inv: Any) -> float | None:
-    # column, old device-log "Total") — the real post-graph-build execution
-    # window, excluding preamble / graph_build / post_orch. Each value is one
-    # launch; the aggregate is over the measured rounds (warmup excluded).
+    _report_effective(stats)
+    return stats
+
+
+def _report_effective(stats: Any) -> None:
+    """Print the ``effective_us (...)`` perf line the daily-CI collector greps.
+
+    The Effective window is the framework's post-graph-build execution window
+    (``orch``∪``sched``, the old device-log "Total"), surfaced directly by
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF002)